chess/interactive.py

```diff
@@ -0,0 +1,187 @@
+
+        while self.__next_move > target + 1:
+            self.__board.pop()
+            self.__next_move -= 1
+
+    def show(self):
+        display(self.__out)
+        next_move = Button(
+            icon="step-forward",
+            layout=Layout(width="60px", grid_area="right"),
+            disabled=self.__next_move >= self.__num_moves,
+        )
+
+        prev_move = Button(
+            icon="step-backward",
+            layout=Layout(width="60px", grid_area="left"),
+            disabled=self.__next_move == 0,
+        )
+
+        reset = Button(
+            icon="stop",
+            layout=Layout(width="60px", grid_area="middle"),
+            disabled=self.__next_move == 0,
+        )
+
+        if self.__next_move == 0:
+            white_move = None
+            black_move = None
+        else:
+            white_move = (
+                self.__white_moves[self.__next_move // 2]
+                if (self.__next_move % 2) == 1
+                else None
+            )
+            black_move = (
+                self.__black_moves[self.__next_move // 2 - 1]
+                if (self.__next_move % 2) == 0
+                else None
+            )
+
+        white_move_list = Select(
+            options=self.__white_moves,
+            value=white_move,
+            rows=max(self.__move_list_len, 24),
+            disabled=False,
+            layout=Layout(width="80px"),
+        )
+
+        black_move_list = Select(
+            options=self.__black_moves,
+            value=black_move,
+            rows=max(self.__move_list_len, 24),
+            disabled=False,
+            layout=Layout(width="80px"),
+        )
+
+        white_move_list.observe(self.__white_select_change)
+        black_move_list.observe(self.__black_select_change)
+
+        move_number_width = 3 + len(str(self.__move_list_len)) * 10
+
+        move_number = Select(
+            options=range(1, self.__move_list_len + 1),
+            value=None,
+            disabled=True,
+            rows=max(self.__move_list_len, 24),
+            layout=Layout(width=f"{move_number_width}px"),
+        )
+
+        move_list = HBox(
+            [move_number, white_move_list, black_move_list],
+            layout=Layout(height="407px", grid_area="moves"),
+        )
+
+        next_move.on_click(self.__next_click)
+        prev_move.on_click(self.__prev_click)
+        reset.on_click(self.__reset_click)
+
+        with self.__out:
+            grid_box = GridBox(
+                children=[next_move, prev_move, reset, self.svg, move_list],
+                layout=Layout(
+                    width=f"{390+move_number_width+160}px",
+                    grid_template_rows="90% 10%",
+                    grid_template_areas="""
+                                "top top top top top moves"
+                                ". left middle right . moves"
+                                """,
+                ),
+            )
+            clear_output()
```

Comment:
```suggestion
            clear_output(wait=True)
```
This will wait to clear the area until the next display is made (reduces flicker)